perf.groovy

```diff
@@ -106,6 +107,10 @@ def static getOSGroup(def os) {
                         batchFile("tests\\runtest.cmd ${configuration} ${architecture} GenerateLayoutOnly")
 
                         // Run with just stopwatch: Profile=Off
+                        if (testEnv != '')
+                        {
+                            batchFile("set TEST_ENV=${testEnv}")
```

Comment:
Unfortunately this will have unintended consequences. This creates a batch file with one line: 

`set TEST_ENV=...`

At the end of the execution the environment is lost and TEST_ENV will be unset for the next batch script called.